### ISSUE TITLE
Keyboard no longer works on Android 9

### ISSUE THREAD

**Issue by Schvenn** [PhilippC/keepass2android#508]
I upgraded to Android pie yesterday and now the keyboard button to the left of the spacebar doesn't provide any options.
Pixel XL 2

**Comment by schrauger** [comment]
I'm having the same problem, only after upgrading to Android pie.
The Essential Phone

**Comment by Schvenn** [comment]
Odd, I would suspect that the keyboard would require the need to display over other apps, because that would likely fix this problem, but it doesn't require that permission.

**Comment by PhilippC** [comment]
duplicate of #377
